lib/models/hum_hub.dart

```diff
@@ -39,4 +48,11 @@ class HumHub {
       return RedirectAction.webView;
     }
   }
+
+  static String generateHash(int length) {
+    final random = Random.secure();
+    const characters = '0123456789abcdef';
+    return List.generate(
+        length, (_) => characters[random.nextInt(characters.length)]).join();
+  }

```

Comment:
The procedure that generates a random hash for a header `x-humhub-app-token`
![image](https://user-images.githubusercontent.com/10835179/217911464-9d5f7830-805c-4225-914f-60032c94eafc.png)